src/Html/FormBuilder.php

```diff
@@ -849,11 +848,11 @@ public function getIdAttribute($name, $attributes)
     /**
      * Get the value that should be assigned to the field.
      *
-     * @param  string  $name
-     * @param  string|array  $value
+     * @param  string|null  $name
+     * @param  string|array|int|null  $value
      * @return string|array|null
      */
-    public function getValueAttribute($name, $value = null)
+    public function getValueAttribute(?string $name = null, $value = null)
```

Comment:
@mjauvin no, it was specifically to ensure that the value is populated in the `input` method in all scenarios - for example, the `submit` method usage that spurned this whole PR in the first place. @arvislacis had a `submit` call which had a value but no name for the field, so when it is passed to this method, `name` is null but the value is not, so it returns the value.

Reply:
Not sure I understand your point, the following call should behave the same whether the method signature has "$name = null" or not, shouldn't it ?
```
getValueAttribute(null, "a Value");
```

Reply:
Yes, but PHPStan would whinge about it, because of [this block of code](https://github.com/wintercms/storm/pull/144/files#diff-ec32bc4f8918dddd6340e4e341849935c6765bdfba774c75b7553376a6109e2eL279-L281) being moved outside of the condition checking if `$name` is not empty.

Reply:
Ahh, gotcha!